app/views/application/personal-details/index.njk

```diff
@@ -76,28 +76,30 @@
     }]
   }) }}
 
-  {{ appAutocomplete({
-    label: {
-      text: "Nationality",
-      classes: "govuk-label--m"
-    },
-    items: nationalities
-  } | decorateApplicationAttributes(["candidate", "nationality"])) }}
-
-  {% set anotherNationality %}
+  {% if data.flags.nationality %}
     {{ appAutocomplete({
       label: {
-        text: "Second nationality"
+        text: "Nationality",
+        classes: "govuk-label--m"
       },
       items: nationalities
-    } | decorateApplicationAttributes(["candidate", "second-nationality"])) }}
-  {% endset -%}
+    } | decorateApplicationAttributes(["candidate", "nationality"])) }}
 
-  {{ govukDetails({
-    summaryText: "Add another nationality",
-    html: anotherNationality,
-    open: ["candidate", "second-nationality"] | ifSetForApplication
-  }) }}
+    {% set anotherNationality %}
+      {{ appAutocomplete({
+        label: {
+          text: "Second nationality"
+        },
+        items: nationalities
+      } | decorateApplicationAttributes(["candidate", "second-nationality"])) }}
+    {% endset -%}
+
+    {{ govukDetails({
+      summaryText: "Add another nationality",
+      html: anotherNationality,
+      open: ["candidate", "second-nationality"] | ifSetForApplication
```

Comment:
What does `ifSetForApplication` do? I’d not spotted this before.

Reply:
I haven't changed it here, just whitespace changes.

It's probably redundant:
```nunjucksAppEnv.addFilter('ifSetForApplication', (sections) => {
      return !!getApplicationValue(sections)
    })```